tests/SymfonyMocks/Kernel.php

```diff
@@ -1,87 +0,0 @@
-<?php
```

Comment:
@andig basically I copied all the code in the previous Kernel mock and put it inside the newly created controllers, that allows us to keep the assertions the same